Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -344,20 +342,15 @@ public void stopDriveMotors() {
     }
 
     /**
-     * turns the LEDs green if it detects the top of a junction, otherwise, turn the lights pink
-     *
+     * turns the LEDs green if the grabber camera detects the top of a junction
+     * otherwise the LEDs stay pink
+     * with both colors, the LEDs are in the "shot" pattern
      */
     public void driveLEDs() {
-        if (blinkinChassis != null) {
-            if (grabberCameraPipeline.detected) {
-                blinkinChassis.setPattern(RevBlinkinLedDriver.BlinkinPattern.GREEN);
-                telemetry.addLine("Detected");
-            } else {
-                blinkinChassis.setPattern(RevBlinkinLedDriver.BlinkinPattern.HOT_PINK);
-                telemetry.addLine("Not Detected");
-            }
+        if (grabberCameraPipeline.detected) {
+            blinkinChassis.setPattern(RevBlinkinLedDriver.BlinkinPattern.CP2_SHOT);
         } else {
-            telemetry.addLine("blinkin chassis null smh");
+            blinkinChassis.setPattern(RevBlinkinLedDriver.BlinkinPattern.CP1_SHOT);
```

Comment:
How does it know what color to set if you deleted the color part?

Reply:
You can set the colors directly from the rev blinking thing in hardware. The software only gives you a limited number of preset colors so that's why the CP1 and CP2 colors are there.